packages/web5/lib/src/dids/data_models/resolution_result.dart

```diff
@@ -1,6 +1,6 @@
-import 'package:web5/src/dids/did_document.dart';
-import 'package:web5/src/dids/did_document_metadata.dart';
-import 'package:web5/src/dids/did_resolution_metadata.dart';
+import 'package:web5/src/dids/data_models/did_document.dart';
+import 'package:web5/src/dids/data_models/did_document_metadata.dart';
+import 'package:web5/src/dids/data_models/resolution_metadata.dart';
```

Comment:
`data_models` feels like a strange name, but maybe it makes sense in this context. Would just `models` work?

Reply:
done: https://github.com/TBD54566975/web5-dart/commit/fa5dbae2693ba6d8664b74fffff21fbb83807b1e

Reply:
totally agreed. changed to `structures`: https://github.com/TBD54566975/web5-dart/commit/f358533a6bd74b0aea1e0297999e5a71c8c2b825